### ISSUE TITLE
DrawNode::drawPoint uses the last color

### ISSUE THREAD

**Issue by ledenis** [cocos2d/cocos2d-x#8963]
When using DrawNode to draw points, it uses the color of the last point.

The bug can be reproduced by putting this code in cpp-tests, DrawPrimitivesTest.cpp in DrawNodeTest():

```
draw->drawPoint(Vec2(200 + 0, 200), 4, Color4F(1, 0, 0, 1));
draw->drawPoint(Vec2(200 + 10, 200), 4, Color4F(0, 1, 0, 1));
draw->drawPoint(Vec2(200 + 20, 200), 4, Color4F(0, 0, 1, 1));
```

The three points are blue, instead of red / green / blue.

Tested as a win32 application.

**Comment by ledenis** [comment]
I also found out that the **pointSize** is affected as well.

**Comment by dabingnn** [comment]
`Color` problem has been fixed,`PointSize` problem is duplicated with #8844
